docs/UserGuide.adoc

```diff
@@ -128,28 +155,16 @@ If the command specifies `f/FRONT, b/BACK`, `b/BACK f/FRONT` is also acceptable.
 
 To access this help guide at any time, type `help` into the command box and press kbd:[Enter].
 
+[[ManagingCards]]
 === Managing Cards
 
-// tag::selectcard[]
-==== Selecting a flashcard : `selectc`
-
-You can select a card that is currently being listed with the `selectc` command. +
-Format: `selectc INDEX`
-
-The following figure shows where the index of a card can be found at:
-
-image::cardIndex.png[width="350"]
-
-Examples:
-
-* `selectc 2` +
-Selects the 2nd card in the list.
-* `select 1` +
-Selects the 1st card in the list.
+The very first step to using Flashy is to populate Flashy with flashcards.
+After which we will proceed to using it to increase your mental capabilities.
 
-// end::selectcard[]
+Managing Card section contains all the information you need to add, select, edit cards and more.
```

Comment:
Can we change this to `This section`? Sounds more grammatically sound.

Reply:
kk